src/Compilers/CSharp/Portable/Microsoft.CodeAnalysis.CSharp.csproj

```diff
@@ -60,7 +60,7 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.CSharp.CommandLine.UnitTests" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.CSharp.Emit.UnitTests" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.CSharp.WinRT.UnitTests" />
-    <InternalsVisibleTo Include="Roslyn.Compilers.CSharp.EnC.UnitTests" />
+    <InternalsVisibleTo Include="Roslyn.Compilers.CSharp.EnC.UnitTests" WorkItem="https://github.com/dotnet/roslyn/issues/35071" />
```

Comment:
I'm guessing these can just be deleted. @tmat?

Reply:
It wasn't obvious from the source control history, so I left it for a follow-up PR to avoid delaying this one.